lib/apr/views/commerce/commerce_order_slack_view.ex

```diff
@@ -14,10 +14,10 @@ defmodule Apr.Views.CommerceOrderSlackView do
   
   def render(subscription, event, routing_key) do
     case {subscription.theme, event["verb"], event["properties"]} do
-      {"fraud", "submitted", %{"mode" => "buy", "items_total_cents" => cents}} when cents >= 3_000_00 ->        
+      {"fraud", "submitted", %{"mode" => "buy", "items_total_cents" => cents}} ->        
         generate_slack_message(event, routing_key)
-      {"fraud", "approved", %{"mode" => "offer", "currency_code" => currency_code, "items_total_cents" => cents}} 
-        when cents >= 9_500_00 and (currency_code == "EUR" or currency_code == "GBP") ->
```

Comment:
Removing the currency check seems fine for now, given that we only support USD, EUR and GBP now. In the future we might need different threshold for different currencies.